examples/simple/simple.c

```diff
@@ -11,7 +11,17 @@
 #include <stdio.h>
 #include <stdlib.h>
 
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
+#define WIN32_EXTRA_LEAN
+#include <windows.h>
+static void ohmd_sleep(double seconds)
+{
+	Sleep((DWORD)(seconds * 1000));
+}
+#else
 void ohmd_sleep(double);
```

Comment:
I feel like the right fix here is just export a ohmd_sleep function in the core API other then this hack. We should inhibit the amount of exported symbols on Linux as well.

Reply:
Both done.